ui/StatusQ/src/onboarding/enums.h

```diff
@@ -42,7 +41,8 @@ class OnboardingEnums
         Locked,
         // exit states
         NotEmpty,
-        Empty
+        Empty,
+        Authorized
```

Comment:
When is this used?

Reply:
We need to call Authorize on the card before doing anything that wants to access the actual data of the card. That includes getting a mnemonic or the keys.